media-sound/zyn-fusion/files/zyn-fusion-gcc10.patch

```diff
@@ -0,0 +1,11 @@
+--- mruby-zest-build-3.0.5/src/mruby-widget-lib/src/gem.c.old	2020-08-11 21:26:06.146444612 +0900
++++ mruby-zest-build-3.0.5/src/mruby-widget-lib/src/gem.c	2020-08-11 21:27:51.839394549 +0900
+@@ -290,7 +290,7 @@
+     return self;
+ }
+ 
+-const struct mrb_data_type mrb_nvg_context_type;
++extern const struct mrb_data_type mrb_nvg_context_type;
```

Comment:
Do you know if this has already been PRed upstream? Arch is carrying a similar patch https://aur.archlinux.org/cgit/aur.git/tree/gcc10.patch?h=zyn-fusion

Reply:
Not sure; this is a common theme for gcc10 compat issues and I wouldn't be surprised if upstream hasn't heard of it yet. I can throw a PR over at them :)

Reply:
OK. Would be nice to do that. Don't need to wait for it with this PR though :)